plugins/org.jkiss.dbeaver.ext.mssql.ui/OSGI-INF/l10n/bundle_tw.properties

```diff
@@ -1,9 +1,9 @@
 # Translated By 2023 Haowei Hsu (hwhsu1231@gmail.com)
-sqlserver.dialog.connection.header = SQL Server\u9023\u7DDA\u8A2D\u5B9A
+sqlserver.dialog.connection.header = SQL Server連線設定
 
-editor.source.name =\u4F86\u6E90
-editor.source.description =\u4F86\u6E90\u78BC
+editor.source.name =來源
+editor.source.description =來源碼
 
-tree.databaseInfo.node.name =\u7BA1\u7406
-tree.sessions.node.name =\u6703\u8A71\u6578
-editor.session_manager.name =\u6703\u8A71
\ No newline at end of file
+tree.databaseInfo.node.name =管理
+tree.sessions.node.name =會話數
+editor.session_manager.name =會話
```

Comment:
You can't write hanzi in the property files. They should be converted to ascii format. Please, return it back everywhere.

Reply:
Copy that. Working on the problem you mentioned